example/cli/main.variant

```diff
@@ -50,13 +50,13 @@ option "vendor-config-path" {
 }
 
 imports = [
-  "git::https://git@github.com/cloudposse/atmos@modules/utils?ref=master",
-  "git::https://git@github.com/cloudposse/atmos@modules/shell?ref=master",
-  "git::https://git@github.com/cloudposse/atmos@modules/kubeconfig?ref=master",
-  "git::https://git@github.com/cloudposse/atmos@modules/terraform?ref=master",
-  "git::https://git@github.com/cloudposse/atmos@modules/helmfile?ref=master",
-  "git::https://git@github.com/cloudposse/atmos@modules/helm?ref=master",
-  "git::https://git@github.com/cloudposse/atmos@modules/workflow?ref=master",
-  "git::https://git@github.com/cloudposse/atmos@modules/istio?ref=master",
-  "git::https://git@github.com/cloudposse/atmos@modules/vendor?ref=master"
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/utils?ref=master",
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/shell?ref=master",
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/kubeconfig?ref=master",
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/terraform?ref=master",
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/helmfile?ref=master",
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/helm?ref=master",
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/workflow?ref=master",
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/istio?ref=master",
+  "git::https://git@github.com/cloudposse/atmos@atmos/modules/vendor?ref=master"
```

Comment:
@Nuru said this change was made because:

> The build copies the entire source folder into the binary, so the source folder should be as small as possible.

But why does `atmos/` not get included, but `modules/` get included?

Reply:
`atmos` gets included. The issue before was that `main.variant` was at the top level, so everything in the repo was being included in the build. Now `main.variant` is in `atmos/`, which only includes the Variant code.